extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ParquetFileWriter.java

```diff
@@ -3,8 +3,8 @@
 //
 package io.deephaven.parquet.base;
 
+import org.apache.commons.io.output.CountingOutputStream;
```

Comment:
Is `com.google.common.io.CountingOutputStream` available currently? If so, that might be the preferable impl. Also, looks like it's long based instead of int based, which may be needed for files > 4GiB?